Source/Blazorise/Layout.razor.cs

```diff
@@ -14,7 +14,9 @@ public partial class Layout : BaseComponent
 
         private bool sider;
 
-        private bool bodyClassApplied;
+        private bool loading;
+
+        private string loadingClass = "b-layout-loading-spinner";
```

Comment:
Make this as ClassBuilder.

Reply:
Will do, was just doing this quickly to get feedback on templating idea :)